ecs-cli/modules/clients/aws/cloudformation/sds_template.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//  http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cloudformation
+
+func GetSDSTemplate() string {
+	return sds_template
+}
+
+var sds_template = `
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "AWS CloudFormation template to create a private dns namespace to enable ECS Service Discovery.",
+  "Parameters": {
+    "SDSDescription": {
+      "Type": "String",
+      "Description": "The description of the private dns namespace",
+      "Default": "Created by the Amazon ECS CLI"
+    },
+    "SDSName": {
+      "Type": "String",
+      "Description": "The name of the Service Discovery Service",
+      "Default": ""
+    },
+    "NamespaceID": {
+      "Type": "String",
+      "Description": "The ID of the namespace that you want to use for DNS configuration",
+      "Default": ""
+    },
+    "DNSType": {
+      "Type": "String",
+      "Description": "The DNS type of the record that you want Route 53 to create.",
+      "Default": ""
```

Comment:
Default is A record, no?

Reply:
It depends on the network mode defined for the current ECS CLI Project. Its A for `awsvpc` tasks and SRV for others.

Reply:
I see. Is this something we will be setting elsewhere then in our code?

Reply:
Yeah. In the code we will set the value of this parameter.